internal/bminventory/inventory.go

```diff
@@ -767,6 +767,17 @@ func (b *bareMetalInventory) createAndUploadNodeIgnition(ctx context.Context, cl
 		return errors.Errorf("Failed to create ignition string for cluster %s", cluster.ID)
 	}
 
+	// Apply overrides to the ignition
```

Comment:
Minor comment but i don't think that this comment is needed, the code here is clear, personally i prefer comments that explain why and not how or what